ultimatepython/classes/encapsulation.py

```diff
@@ -82,24 +91,25 @@ def main():
     # Check if the accounts are an instance of the BankAccount class.
     assert all(isinstance(account, BankAccount) for account in accounts)
     # Check if the account balance are floats.
-    assert all(isinstance(account.get_balance(), float) for account in accounts)
+    assert all(isinstance(account.get_balance(), int) for account in accounts)
     # Check if the account balance are integers.
     assert all(isinstance(account.get_account_number(), int) for account in accounts)
 
     # Deposit amount and check if the balance is updated.
-    account1.deposit(100.0)
-    assert (account1.get_balance() == 100.0)
+    account1.deposit(100)
+    assert (account1.get_balance() == 100)
 
     # Withdraw amount and check if the balance is updated.
-    account1.withdraw(50.0)
-    assert (account1.get_balance() == 50.0)
+    account1.withdraw(50)
+    assert (account1.get_balance() == 50)
 
     # Assert the data types of account balance.
-    assert (isinstance(account1.get_balance(), float))
+    assert (isinstance(account1.get_balance(), int))
```

Comment:
Is this assert really needed? I thought we are using only `int` in this module

Reply:
I have removed them in the latest commit.